pkg/trace/api/api_test.go

```diff
@@ -368,6 +368,42 @@ func TestReceiverDecodingError(t *testing.T) {
 	})
 }
 
+func TestReceiveEndpointSniffingRequests(t *testing.T) {
+	assert := assert.New(t)
+	conf := newTestReceiverConfig()
+	r := newTestReceiverFromConfig(conf)
+	server := httptest.NewServer(http.HandlerFunc(r.handleWithVersion(v05, r.handleTraces)))
```

Comment:
Your `v04` test fails because that endpoint is not registered. You should maybe replace this line with:
```suggestion
	go r.Start()
	defer r.Stop()
```

Reply:
Great, thanks